internal/slicer/slicer.go

```diff
@@ -397,3 +391,63 @@ func createFile(targetPath string, pathInfo setup.PathInfo) (*fsutil.Entry, erro
 		MakeParents: true,
 	})
 }
+
+// selectPkgArchives selects the appropriate archive for each selected slice
+// package. It returns a map of archives indexed by package names.
+func selectPkgArchives(archives map[string]archive.Archive, selection *setup.Selection) (map[string]archive.Archive, error) {
+	// Sort the archives by their priority value.
+	releaseArchives := make([]*setup.Archive, 0, len(selection.Release.Archives))
+	for _, archive := range selection.Release.Archives {
+		if archive.Priority < 0 {
+			// Ignore negative priority archives unless a package specifically
+			// asks for it with the "archive" field.
+			continue
+		}
+		releaseArchives = append(releaseArchives, archive)
+	}
+	slices.SortFunc(releaseArchives, func(a, b *setup.Archive) int {
+		if a.Priority > b.Priority {
+			return -1
+		}
+		if a.Priority < b.Priority {
+			return 1
+		}
+		return 0
```

Comment:
I think `b.Priority - a.Priority` works.

Reply:
So, I did that first and then I thought about a possible `int` overflow. If the two values are on the opposite end of the limits, the diff would overflow the size.

Reply:
TODO: add a maximum value, sanitize the input first and then do the subtraction check.

Reply:
Done in https://github.com/letFunny/chisel/pull/12/commits/1f4a4545e7dfbde0d5309b2a3ed4474a28afd460.